packages/cdktf-cli/package-lock.json

```diff
@@ -1,892 +0,0 @@
-{
```

Comment:
Is this by accident?

Reply:
It had been committed by accident by me, when I built the watch feature 😅